package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "bootstrap": "^5.1.3",
+    "react-bootstrap": "^2.0.2"
```

Comment:
Remove this file. You probably accidentally ran `yarn` in the parent directory of our repo, instead of in client/.

Reply:
Remove the package-lock.json file as well.